app/jobs/calendar_importer/parsers/dice_fm.rb

```diff
@@ -5,53 +5,25 @@
 # Parent parser classes should not be added.
 
 module CalendarImporter::Parsers
-  class DiceFm < Base
-    NAME = 'Dice FM'
+  class DiceFm < LdJson
+    NAME = 'Dice FM (LD+JSON)'
     KEY = 'dice-fm'
     DOMAINS = %w[dice.fm].freeze
 
     def self.whitelist_pattern
```

Comment:
tiny note but we should swap `whitelist` for `allowlist`, not sure if part of this pr or a new one